oak/proto/label.proto

```diff
@@ -66,16 +66,7 @@ message WebAssemblyModuleTag {
 //
 // Applies to both the HTTP and gRPC client pseudo-nodes.
 message TlsEndpointTag {
-  // The Subject Alternative Name (SAN) of a certificate that a remote endpoint must be able to
-  // present in order to be allowed to receive data with this confidentiality tag.
-  //
-  // In general a certificate may have multiple SANs; an HTTP or gRPC client pseudo-node connected
-  // to a remote endpoint with multiple SANs is able to declassify data with a TlsEndpointTag
-  // referring to any of these SANs.
-  //
-  // The certificate is validated by the Oak Runtime using the set of Certificate Authorities (CA)
-  // that are available to it.
-  //
-  // See https://tools.ietf.org/html/rfc5280#section-4.2.1.6
-  string certificate_subject_alternative_name = 1;
+  // The TLS authority (host:port) of the remote endpoint, which is validated by the Oak Runtime
```

Comment:
nit: I think host is sufficient for trust. Port is probably not needed.

Reply:
Yes I think you are right, but since we have the port, it may help prevent some confused deputy problem in which someone manages to get a web page on some third party server that also has some gRPC server running, and get data out. Not entirely sure, we can easily take it out in the future if we think it's safe anyways.